WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -206,7 +208,8 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleDoubleTapCoordinate:(FBRouteRequest *)request
 {
-  XCUICoordinate *doubleTapCoordinate = [self.class getGestureCoordinate:request];
+  CGPoint doubleTapPoint = CGPointMake((CGFloat)[request.arguments[@"x"] doubleValue], (CGFloat)[request.arguments[@"y"] doubleValue]);
+  XCUICoordinate *doubleTapCoordinate = [self.class gestureCoordinateWithCoordinate:doubleTapPoint application:request.session.application shouldApplyOrientationWorkaround:YES];
```

Comment:
Because of the same issue that I've already described. We never know how these coordinates were calculated by the client - based on the absolute screen width/height or based on the element position. But nevermind - I will add this check since we have included a fix for screen size into this PR.